code/src/js components/Questionamountofcoffee.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+export const Questionamountofcoffee = ({answeramount, onAmountChange}) => {
+return(
+<>
+    <div>
+        <p tabIndex='0' >How many cups of coffee do you drink per week?</p>
+            <label tabIndex='0' htmlFor="answerAmount">1-3</label>
+                <input type="radio"
+                onChange = {event => onAmountChange(event.target.value)}
+                value = "1-3"
+                name="cups"
+                checked = {answeramount === "1-3"}
+                required
+                aria-label="Select the amount of cups of coffee you drink per week"
+            />
```

Comment:
Try to be consistent with the first "value" after the element name. Sometimes you keep it in the same line as the element (like you've done here). Sometimes you put in on a new row (like you've done on the select-element in Questiondrink.js). I think that you should put it in a new row all the time.